deployment-examples/gcp/README.md

```diff
@@ -24,12 +22,12 @@ Google Cloud Project. It requires to have:
 |`GCLOUD_CLUSTER_NAME` | Existing Google Kubernetes Engine cluster name | YES | `adyen-demo` |
 |`GCLOUD_KMS_KEYRING` | Google KMS [key-ring](https://cloud.google.com/kms/docs/resource-hierarchy#key_rings) which include the encrypted key for secret.yaml files | YES | `adyen-integration-deployment-demo` |
 |`GCLOUD_KMS_KEY_NAME` | Google KMS key name which used to encrypt the secret.yaml files | YES | `adyen-extension-module` |
-|`TAG` | Indented [release version](https://github.com/commercetools/commercetools-adyen-integration/releases) of commercetools-adyen-integration | YES | `v5.0.0` |
+|`TAG` | Indented [release version](https://github.com/commercetools/commercetools-adyen-integration/releases) of commercetools-adyen-integration | YES | `x.x.x` |
 |`HELM_CHARTS_REPO` | Github repository URL for helm charts | NO | `https://github.com/commercetools/k8s-charts.git` |
 |`HELM_CHARTS_VERSION` | Intended release version of the helm charts repository | YES | `1.7.5` |
 |`ENVIRONMENT_NAME` | Folder name that contains secrets.yaml file in `extension/k8s` and `notification/k8s` | YES | `demo` |
 
-After configured all the required values, execute the `deploy-to-gcp.sh` script file.
+After configured all the required values, execute the [`deploy-to-gcp.sh`](deploy-to-gcp.sh) script file.
 ```
```

Comment:
would you put the EXPORT statements for variables here to make it clear visually ( also could save time for the devs who would run the script) for the env variables? the table is something for descriptions imo :)

Reply:
655af40a18f0b6b367f0a5011691acb3afbcb347

Reply:
I'm not sure if this is what I meant.

Reply:
fb884884da655d3f3dbd37070c836f9fa3449d5d